rules/lfi-os-files.data

```diff
@@ -199,1141 +272,267 @@ etc/libnl-3
-var/log/nginx/error_log
-var/log/nginx/access.log
-var/log/nginx/error.log
-var/log/nginx.access_log
-var/log/nginx.error_log
-var/log/samba/log.smbd
-var/log/samba/log.nmbd
-var/log/samba.log
-var/log/samba.log1
-var/log/samba.log2
-var/log/log.smb
-var/log/ipfw.log
-var/log/ipfw
-var/log/ipfw/ipfw.log
-var/log/ipfw.today
-var/log/poplog
-var/log/authlog
-var/log/news.all
-var/log/news/news.all
-var/log/news/news.crit
-var/log/news/news.err
-var/log/news/news.notice
-var/log/news/suck.err
-var/log/news/suck.notice
-var/log/messages
-var/log/messages.1
-var/log/user.log
-var/log/user.log.1
-var/log/auth.log
-var/log/pm-powersave.log
-var/log/xorg.0.log
-var/log/daemon.log
-var/log/daemon.log.1
-var/log/kern.log
-var/log/kern.log.1
-var/log/mail.err
-var/log/mail.info
-var/log/mail.warn
-var/log/ufw.log
-var/log/boot.log
-var/log/syslog
-var/log/syslog.1
-var/log/squirrelmail.log
-var/log/apache2/squirrelmail.log
-var/log/apache2/squirrelmail.err.log
-var/log/mail.log
-var/log/vmware/hostd.log
-var/log/vmware/hostd-1.log
-# Wordpress
-wp-config.php
-wp-config.bak
-wp-config.old
-wp-config.temp
-wp-config.tmp
-wp-config.txt
-# Symfony
-config.yml
-config_dev.yml
-config_prod.yml
-config_test.yml
-parameters.yml
-routing.yml
-security.yml
-services.yml
-# Drupal
-sites/default/default.settings.php
-sites/default/settings.php
-sites/default/settings.local.php
-# Magento
-app/etc/local.xml
-# Sublime Text
-sftp-config.json
-# ASP.NET
-Web.config
-# vBulletin
-includes/config.php
-# OSCommerce
-includes/configure.php
-# phpMyAdmin
-config.inc.php
-# MediaWiki
-LocalSettings.php
-# MyBB
-inc/config.php
-# TYPO3
-typo3conf/localconf.php
-# Laravel
-# Note: these entries might be benign in REQUEST_FILENAME
-config/app.php
-config/custom.php
-config/database.php
-# Joomla
-# Note: this string might be benign in REQUEST_FILENAME
-/configuration.php
-# phpBB
-# Note: this string might be benign in REQUEST_FILENAME
-/config.php
-#Missing Debian based sensitive directories
-var/mail/www-data
-etc/network/
-etc/init/
-#IIS
-inetpub/wwwroot/global.asa
-system32/inetsrv/config/applicationhost.config
-system32/inetsrv/config/administration.config
-system32/inetsrv/config/redirection.config
-#Windows files
-system32/config/default
-system32/config/sam
-system32/config/system
-system32/config/software
-winnt/repair/sam._
-# Node
-package.json
-package-lock.json
-gruntfile.js
-npm-debug.log
+opt/lsws/logs
+opt/nuxeo
+opt/tomcat
+opt/xampp
 ormconfig.json
-tsconfig.json
-webpack.config.js
-yarn.lock
-# /proc entries (keep in sync with restricted-files.data)
+package-lock.json
+package.json
+parameters.yml
+pgsql/bin/pg_passwd
+pgsql/data
+php/apache.conf
+php/apache.conf.php
+php/apache2.conf
+php/httpd.conf
+php/lib/php.ini
+php/php.ini
+php5/apache.conf
+php5/apache2.conf
+php5/httpd.conf
+php5/lib/php.ini
+php5/php.ini
+php7/php.ini
+php8/php.ini
+postgresql/log/pgadmin.log
+private/etc/httpd
+private/etc/squirrelmail/config/config.php
 proc/0
```

Comment:
I think it might be ok to block anything under `proc`.

Reply:
@theseion considering the rule 930120 is a PL1 rule, and that this will be used with `REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES_NAMES|ARGS_NAMES|ARGS|XML:/*`, do you think we should be only using `proc`?